src/MiniExcel/OpenXml/ExcelOpenXmlSheetWriter.Async.cs

```diff
@@ -168,6 +185,8 @@ private static async Task WriteDimensionAsync(MiniExcelAsyncStreamWriter writer,
             writer.SetPosition(position);
         }
 
+        // Async and sync diverge. Do not apply the attribute
+        //[Zomp.SyncMethodGenerator.CreateSyncVersion]

```

Comment:
@michelebastione this diverged from the from the sync version as the result of a5c3e88, so not applying the attribute yet. Please advice that the sync version should be copied to the async one.